packages/lesmis-server/src/app/createApp.ts

```diff
@@ -46,6 +47,7 @@ export function createApp() {
   const app = new OrchestratorApiBuilder(new TupaiaDatabase(), 'lesmis')
     .useSessionModel(LesmisSessionModel)
     .useAttachSession(attachSession)
+    .attachApiClientToContext(authHandlerProvider) // after useAttachSession
```

Comment:
Not sure if this comment `after useAttachSession` is necessary.. But it took me a while to figure it out.